descriptions/SDK.Expat.md

```diff
@@ -0,0 +1 @@
+[**Expat**](https://libexpat.github.io/) is a stream-oriented XML parser.
```

Comment:
```suggestion
[**Expat**](https://libexpat.github.io/) is a stream-oriented XML parser written in C.
```
Their full description is fine

Reply:
didn't see the written in C part on their github repo readme (thats where i took it from) but yeah, wouldn't hurt i guess